src/browser/existing-browser.js

```diff
@@ -35,6 +36,10 @@ module.exports = class ExistingBrowser extends Browser {
     async init({ sessionId, sessionCaps, sessionOpts } = {}, calibrator) {
         this._session = await this._attachSession({ sessionId, sessionCaps, sessionOpts });
 
+        if (!isRunInNodeJsEnv(this._config)) {
+            this._collectCustomCommands();
```

Comment:
Collect custom commands only in browser environment. It is used in order to mock these methods on client side

Reply:
This public field doesn't available for user. User can use only `publicAPI` field from browser. Which can be used as `browser` instance inside test. Or we emit it on `NEW_BROWSER` event - https://github.com/gemini-testing/testplane/blob/master/src/worker/runner/browser-pool.js#L30.

As a result, since this field doesn't public for user and it's not used in any way inside (in nodejs runner), therefore, I don't see the reason to collect them.

We can start collecting them when we really need it. What do you think?